{"name": "Login shows an error message when trying with invalid credentials", "status": "passed", "steps": [{"name": "Given i navigate to the login page", "status": "passed", "start": 1716491274097, "stop": 1716491277687}, {"name": "When i enter invalid credentials and click on sign in button", "status": "passed", "start": 1716491277687, "stop": 1716491278352}, {"name": "Then i should be able to see an invalid email or password message", "status": "passed", "start": 1716491278352, "stop": 1716491278391}], "parameters": [{"name": "username", "value": "'asd'"}, {"name": "password", "value": "'testwise'"}], "start": 1716491274097, "stop": 1716491278391, "uuid": "d7928978-177e-4b7a-9200-c5f785648ea4", "historyId": "a54cc321888a36961e44b96ea0ac4502", "testCaseId": "6485cd1aa206d185bd707f422a9a9238", "fullName": "tests.test_login#test_invalid_login", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Login"}, {"name": "epic", "value": "Web UI"}, {"name": "story", "value": "Invalid login"}, {"name": "tag", "value": "login"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "Julian"}, {"name": "thread", "value": "17804-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}